banzai/main.py

```diff
@@ -155,9 +155,11 @@ def process_directory(pipeline_context, raw_path, image_types=None, log_message=
     if len(log_message) > 0:
         logger.info(log_message, extra_tags={'raw_path': raw_path})
     image_path_list = image_utils.make_image_path_list(raw_path)
-    image_path_list = image_utils.select_images(image_path_list, pipeline_context,
-                                                image_types=image_types, db_address=pipeline_context.db_address)
-    for image_path in image_path_list:
+    if image_types is None:
+        image_types = [None]
+    images_to_reduce = [image_utils.select_images(image_path_list, pipeline_context, image_type)
```

Comment:
`image_utils.select_images()` needs to be updated to handle the `image_type` parameter